seeker/message.html

```diff
@@ -0,0 +1,187 @@
+<!DOCTYPE html>
```

Comment:
Could rename this to messages.html instead, I have it set up that way in the navbar dropdown.

Also, can you add a button which says new message so it can redirect to message-new.html?